### ISSUE TITLE
Multiple users support

### ISSUE THREAD

**Issue by kawasaki** [kawasaki/bluepy-scratch-link#14]
In issue #4, @miguev pointed out that some chore work is required to use bluepy-scratch-link by multiples users. Here I quote his comment.

> It is also necessary for each user to have their own copy of bluepy-scratch-link and run ./gencert.sh in it, and keep ownership fo the bluepy-scratch-link directory.

I open this issue to think about it.

**Comment by kawasaki** [comment]
One idea I can think of is to generate certificate and key files for each user, and keep them in each users home directory (maybe in .config/bluepy-scratch-link/). With this, each user does not need to keep ownership of bluepy-scratch-link directory.

Still each user need to run ./gencert.sh. Another idea is to rewrite this script with python, and integrate into scratch_link.py. Each user just need to run scrach_link.py, and certificate and key files are automatically generated in it.

I will try to implement these ideas.

**Comment by kawasaki** [comment]
I have implemented the ideas. gencert.sh was replaced with gencert.py. scratch_link.py is integrated with gencert.py, then each user no longer need to run gencer.sh. The certificate is generated for each user and stored in home directory of the user. The users no longer need to care where the certificate file is.

The implementation is now in dev branch. After some some more review, will merge to the master branch.
